### DISCUSSION TITLE
Institute issued email problem

### DISCUSSION THREAD

**Discussion by Abdulsattar-cmd** [community/community#54407] [GitHub Education]
### Hi everyone,

Can anybody help me to re verify my student status
I don't have academic email because my university don't provide it I have
Admission latter
Dated fees challan
Student portal
When I try to submit application for student GitHub package for re verify email I am receiving that use academic email but my university is not providing it
How Can i solve it 
Or it is end for me student pack 😭

**Comment by samus-aran** [comment]
Hi @Abdulsattar-cmd 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school. We would recommend to you speak to your institution regarding to acquiring an email address to proceed.
